.github/workflows/infra.yml

```
@@ -23,6 +23,7 @@ jobs:
     - run: npm ci
     - run: npm run build
     - run: npx playwright install-deps
+    - run: npx playwright install
```

```suggestion
    - run: npx playwright install --with-deps
```
nit: remove the file above it.

---

I actually want to see groups for those in the UI!